models/request.js

```diff
@@ -0,0 +1,74 @@
+var User = require('./user');
+var Service = require('./service');
+module.exports = function(sequelize, DataTypes) {
+    var Request = sequelize.define("Request", {
+      recipient_id: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: false,
+        autoIncrement: true,
+        unique: true
+      },
+      services_id: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: false,
+        autoIncrement: true,
+        unique: true
+      },
+      finalized: {
+        type: DataTypes.BOOLEAN,
+        primaryKey:true,
+        allowNull: true,
+      },
+      detail: {
+        type: DataTypes.TEXT,
+        allowNull: false,
+        validate: {
+          len: [1]
+        }          
+      },
+      requester_rating: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: true,
+        autoIncrement: true,
+        unique: true
```

Comment:
This should reflect:
`requester_rating INT`

`primaryKey` and `autoIncrement` are not needed
not sure what `unique` does